rust/ql/lib/codeql/rust/security/HardcodedCryptographicValueExtensions.qll

```diff
@@ -0,0 +1,92 @@
+/**
+ * Provides classes and predicates for reasoning about hardcoded cryptographic value
+ * vulnerabilities.
+ */
+
+import rust
+private import codeql.rust.dataflow.DataFlow
+private import codeql.rust.dataflow.internal.DataFlowImpl
```

Comment:
I think it would be better to import `codeql.rust.dataflow.FlowSource` and `codeql.rust.dataflow.FlowSink` instead of something from `internal`.

Reply:
Done (in a few places, not just this query).